### PR TITLE
Readme: describe `--ignore_chromium` flag

### PR DESCRIPTION
**Pull request by petemill**
Additional information for https://github.com/brave/brave-browser/issues/16419. This will never need to be uplifted.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -165,21 +166,21 @@ brave-browser> npm run sync --create branch_name
 
 ### Update the current branch to latest remote
 ```bash
-brave-core> git pull
+brave-core> git pull # or git reset --hard origin/branch_name
```
Comment (bridiver) [inline README.md:169]: these are not equivalent unless you fetch before reset

Comment (petemill) [reply README.md:153]: true! I'll leave out, this is not a `git` readme after all

Comment (petemill) [reply README.md:153]: I just dislike doing pull when it's not the `master` branch

Comment (bridiver) [reply README.md]: I never use `git pull`, I always fetch/merge/reset/...

**Review by bridiver — APPROVED** [review]

**Pull request merged**